src/components/Announcement.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
Are we using names starting with capital letters?

Reply:
Yeah, so this is the convention being followed for React library-level imports (similar to how we import ReactDOM). So we're sticking to that.

Reply:
Makes sense